helm-charts/falcon-kac/templates/deployment_webhook.yaml

```diff
@@ -359,4 +359,4 @@ webhooks:
     - jobs
     scope: '*'
   sideEffects: None
-  timeoutSeconds: 5
\ No newline at end of file
+  timeoutSeconds: 10
```

Comment:
@pflanagan-cs can you add the End of File newline? Once that is added, it's good to merge.

Reply:
done